docs/08-✅-releases.md

```diff
@@ -36,17 +38,19 @@ In a similar vein as above, the Android internal beta is managed through Google
 
 #### External Beta
 
-We take a slightly different approach to iOS. Due to not being able to determine within the app whether or not the app is in Beta or release, we have to create a release branch. This ensures that what we test in Beta is at the same point in time to what we release. We use the naming convention of `release/[version]` when creating this branch and is branched from Master.
+We take a slightly different approach to iOS. Due to not being able to determine within the app whether or not the app is in Beta or release, we have a different build configuration for releasing to production - which hides the 'report bug' button.
 
-The release number as it stands is `1.0.[next number]`. There is a ticket to improve this in the backlog.
+We build the APK using `android-beta-deploy`. This will release a build to google play to the 'internal beta' group for internal testing by the team. It is then manually promoted within the Google Play console to our external beta testers.
 
-We build the APK using `android-beta-deploy`. This will first go into internal beta where the team will check it. It is then promoted within the Google Play console to our external beta testers.
+As with iOS, releases to the play store can be tracked in [github releases](https://github.com/guardian/editions/releases).
```

Comment:
I think it would be good to make to clear that each internal release creates a github release?